go.mod

```diff
@@ -56,81 +56,81 @@ require (
 )
 
 require (
-	github.com/Code-Hex/go-infinity-channel v1.0.0 // indirect // gomodjail:confined
-	github.com/VividCortex/ewma v1.2.0 // indirect // gomodjail:confined
-	github.com/a8m/envsubst v1.4.2 // indirect // gomodjail:confined
-	github.com/alecthomas/participle/v2 v2.1.1 // indirect // gomodjail:confined
-	github.com/bahlo/generic-list-go v0.2.0 // indirect // gomodjail:confined
-	github.com/bmatcuk/doublestar/v4 v4.7.1 // indirect // gomodjail:confined
-	github.com/braydonk/yaml v0.9.0 // indirect // gomodjail:confined
-	github.com/buger/jsonparser v1.1.1 // indirect // gomodjail:confined
-	github.com/containerd/errdefs v0.3.0 // indirect // gomodjail:confined
-	github.com/containerd/log v0.1.0 // indirect // gomodjail:confined
-	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect // gomodjail:confined
-	github.com/digitalocean/go-libvirt v0.0.0-20220804181439-8648fbde413e // indirect // gomodjail:confined
-	github.com/dimchansky/utfbom v1.1.1 // indirect // gomodjail:confined
-	github.com/djherbis/times v1.6.0 // indirect // gomodjail:confined
-	github.com/elliotchance/orderedmap v1.7.1 // indirect // gomodjail:confined
-	github.com/emicklei/go-restful/v3 v3.11.0 // indirect // gomodjail:confined
-	github.com/fatih/color v1.18.0 // indirect // gomodjail:confined
+	github.com/Code-Hex/go-infinity-channel v1.0.0 // indirect; indirect // gomodjail:confined
```

Comment:
Looks like dependabot doesn't want `// gomodjail:confined` to be appended after `// indirect`.
So we will have to revert the gomodjail policy: Apply `gomodjail:confined` to the global scope, and then apply `gomodjail:unconfined` to a small set of dependencies